core/src/com/unciv/ui/pickerscreens/PantheonPickerScreen.kt

```diff
@@ -17,7 +17,9 @@ class PantheonPickerScreen(
         for (belief in ruleset.beliefs.values) {
             if (belief.type != BeliefType.Pantheon) continue
             val beliefButton = getBeliefButton(belief, withTypeLabel = false)
-            if (choosingCiv.religionManager.isPickablePantheonBelief(belief)) {
+            if (gameInfo.civilizations.none {
+                        it.religionManager.religion?.followerBeliefs?.contains(belief.name) == true
+            }) {
```

Comment:
This is in my opinion a bit of a loss in readability. Maybe replace it with a function `anyCivHasBelief(belief: Belief)` that can then also be used in `chooseBeliefOfType` in `NextTurnAutomation.kt` (which does basically the same computation)?